vassal-app/src/main/java/VASSAL/counters/Labeler.java

```diff
@@ -607,14 +615,60 @@ public BufferedImage eval() throws Exception {
 
       // paint the foreground
       if (fg != null) {
-        final JLabel l = makeLabel();
-        l.paint(g);
+        if (txt.contains("<img")) { //NON-NLS
```

Comment:
Should it have a trailing space, i.e., "<img "?

Reply:
No, and in addition to the aesthetic value of supporting a plain <img> or <img/> tag, I was even realizing that an <img> tag in a string that only *sometimes* contains an image would have the extra value of making both versions run the JTextPane version rather than the JLabel version, so the text would resolve to the same location. (The JLabel and JTextPane versions don't naturally resolve to appearing in quite exactly the same positions as each other, and I have neither the time nor the beer to make them cooperate perfectly, if you know what I mean)